python_modules/libraries/dagster-dbt/dagster_dbt/cloud/resources.py

```diff
@@ -181,6 +181,44 @@ def run_job(self, job_id: int, **kwargs) -> Dict[str, Any]:
         )
         return resp
 
+    def get_runs(
+        self,
+        include_related: Optional[List[str]] = [],
+        job_definition_id: Optional[int] = None,
+        order_by: Optional[str] = "-id",
+        offset: int = 0,
+        limit: int = 100,
+    ) -> List[Dict[str, any]]:
+        """
+        Returns a list of runs from dbt cloud. This can be optionally filtered to a specific job
```

Comment:
tiny nitpick, but elsewhere, it's "dbt Cloud" not "dbt cloud"

Reply:
👍

Reply:
resolved this and the instances where I wrote `DBT Cloud`

Reply:
resolved this and the instances where I wrote `DBT Cloud`